task/ko/0.1/ko.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: ko
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/categories: Image Build
+    tekton.dev/tags: image-build
+    tekton.dev/displayName: "Build and upload container image using ko"
+    tekton.dev/platforms: "linux/amd64,linux/arm64"
+spec:
+  description: >-
+    This Task builds source into a container image using ko.
+  params:
+    - name: ko-image
+      description: ko image to use for builds
+      default: "ghcr.io/ko-build/ko:latest"
+    - name: main
```

Comment:
In ko's jargon, it uses the `importpath` keyword for the path that includes the main package. What do you think about replacing it with `importpath`?

Reply:
I'm a skaffold user, so I used `main` to match https://skaffold.dev/docs/pipeline-stages/builders/ko/#source-file-locations